tox.ini

```diff
@@ -30,5 +30,14 @@ commands = {posargs}
 # W504 ignore wrap line after binary operator
 ignore = E501,W504
 show-source = True
-exclude = ./.*,build,dist,*egg
+exclude = ./.*,build,dist,*egg,venv
 import-order-style = pep8
+
+[testenv:coverage]
```

Comment:
After running `tox -e coverage`, I've got some untracked unignored files in the repo:

![image](https://user-images.githubusercontent.com/3383210/130430165-4061f993-32a3-4399-b784-d274ea1fef3d.png)

I think they should be added to .gitignore, and also, less obviously, as `tar-ignore` to `debian/source/options` (so that they're not included in Debian/Ubuntu packages)